Docs/docs/worlds/components/vrc_scenedescriptor.md

```diff
@@ -32,3 +32,17 @@ Describes your VRChat world. Required for every Unity scene you'd like to use as
 | Unity Version                | Unity version being used; you should never need to touch this.                                                  |
 | Dynamic Prefabs              | An array of prefabs that can be dynamically spawned at runtime. Any prefab that isn't referenced in the scene but needs to be spawned should be listed here. |
 | Dynamic Materials            | An array of materials that can be dynamically changed at runtime. Any material that isn't referenced in the scene but needs to be set to an object should be listed here. |
+
+<br/>
+<!-- Description of Physics Matrix here -->
+<h3>Interaction Block and Passthrough on VRChat Layers</h3>
+Interaction (grabbing an item from a distance, toggling a UI element with the laser) is blocked by most VRChat layers. The following layers are transparent to interaction and allow you to interact through them:
+<ul>
+<li>UiMenu</li>
+<li>UI</li>
+<li>PlayerLocal</li>
+<li>MirrorReflection</li>
+</ul>
```

Comment:
We prefer Markdown instead of HTML for compatibility, so I've suggested the equivalent markup code below:
```suggestion

### Interaction Block and Passthrough on VRChat Layers

Interaction (grabbing an item from a distance, toggling a UI element with the laser) is blocked by most VRChat layers. The following layers are transparent to interaction and allow you to interact through them:

- UiMenu
- UI
- PlayerLocal
- MirrorReflection
```